commands/apps.go

```diff
@@ -799,13 +800,23 @@ func RunAppsConsole(c *CmdConfig) error {
 	}
 	appID := c.Args[0]
 	component := c.Args[1]
+	var instanceID string
+	if len(c.Args) >= 3 {
+		instanceID = c.Args[2]
+	}
 
 	deploymentID, err := c.Doit.GetString(c.NS, doctl.ArgAppDeployment)
 	if err != nil {
 		return err
 	}
 
-	execResp, err := c.Apps().GetExec(appID, deploymentID, component)
+	opts := &godo.AppGetExecOptions{
+		DeploymentID: deploymentID,
+		Component:    component,
+		InstanceID:   instanceID,
+	}
+	execResp, err := c.Apps().GetExecWithOpts(appID, opts)
```

Comment:
Since component name is a required field, I think it would be more ergonomic to make that an actual argument to `GetExecWithOpts` (outside the AppGetExecOptions struct). WDYT?

```suggestion
	opts := &godo.AppGetExecOptions{
		DeploymentID: deploymentID,
		InstanceID:   instanceID,
	}
	execResp, err := c.Apps().GetExecWithOpts(appID, component, opts)
```